CriteoPublisherSdk/Sources/Logging/CR_LogHandler.m

```diff
@@ -0,0 +1,103 @@
+//
+//  CR_LogHandler.m
+//  CriteoPublisherSdk
+//
+//  Copyright © 2018-2020 Criteo. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+#import "CR_LogHandler.h"
+
+static const CR_LogSeverity crConsoleSeverityThresholdDefault = CR_LogSeverityWarning;
+
+@implementation CR_ConsoleLogHandler
+
+@synthesize severityThreshold = _severityThreshold;
+
+#pragma mark - Lifecycle
+
+- (instancetype)init {
+  if (self = [super init]) {
+    self.severityThreshold = crConsoleSeverityThresholdDefault;
```

Comment:
Question:
Here, your setting the property rather than its backing field `_severityThreshold`.
Do you have a documentation explaining when to set the property, when to set the backing field, and what is the difference ?

Reply:
The coding style we are following has a [chapter about this topic](https://github.com/raywenderlich/objective-c-style-guide#underscores):
> An exception to this: inside initializers, the backing instance variable (i.e. _variableName) should be used directly to avoid any potential side effects of the getters/setters.

Amending then.